src/utils/tests/limitNumberOfCharacters.test.ts

```diff
@@ -0,0 +1,35 @@
+import { describe, it, vi } from 'vitest';
+
+import { cleanup } from '@testing-library/react';
+
+import { limitNumberOfCharacters } from '../limitNumberOfCharacters';
+
+describe(`limitNumberOfCharacters`, () => {
+  afterAll(() => {
+    vi.clearAllMocks();
+  });
+
+  afterEach(() => {
+    cleanup();
+    vi.resetAllMocks();
+  });
+
+  it(`String less than 108 characters must be returned unchanged`, () => {
+    const input = `Lorem ipsum dolor.`;
+    const result = limitNumberOfCharacters(input);
+    expect(result).toBe(input);
+  });
+
+  it(`String with exactly 108 characters must be returned unchanged`, () => {
+    const input = `Lorem ipsum dolor sit amet consectetur adipisicing elit.`;
```

Comment:
```suggestion
    const input = `Lorem ipsum dolor sit amet consectetur adipisicing elit. Ea distinctio consequuntur consequatur repudiandae.`;
```

there is 56 characters

Reply:
Good observation, done!